src/components/TracksTable.vue

```diff
@@ -88,7 +88,11 @@
         <TrackGenres :track="props.item" />
       </template>
       <template v-slot:item.play_count="props">
-        {{ playCountsByTrack[props.item.id] || 0 }}
+        {{
+          (playStatsByTrack[props.item.id] &&
+            playStatsByTrack[props.item.id].count) ||
+          0
```

Comment:
```suggestion
          playStatsByTrack[props.item.id]?.count || 0
```
Or does this not work in the templates?

Reply:
This does not work in templates